<launch>
    <node pkg="hole_detection" type="detection" name="detectionFront" output="screen">
        <!-- topics and frames -->
        <param name="subscriber_topic" type="str" value="/firefly/vi_sensor/camera_depth/depth/points" />
        <param name="camera_frame_topic" type="str" value="firefly/vi_sensor/camera_depth_optical_center_link" />
        
        <!-- camera params -->
        <param name="front" type="bool" value="true" />
        <param name="mynteye" type="bool" value="false" />

        <!-- templaye path -->
        <param name="manhole_path" type="string" value="$(find hole_detection)/resource/manhole.txt"/>
        <param name ="big_hole_path" type="string" value="$(find hole_detection)/resource/bigHole.txt"/>
        
        <!-- debug -->
        <param name="debug_vis" type="bool" value="false" />
    </node>

    <!-- <node pkg="hole_detection" type="detection" name="detectionDown" output="screen">
        <param name="subscriber_topic" type="str" value="/firefly/vi_sensor_down/camera_depth/depth/points" />
        <param name="camera_frame_topic" type="str" value="firefly/vi_sensor_down/camera_depth_optical_center_link" />
        <param name="debug_vis" type="bool" value="false" />
        <param name="front" type="bool" value="false" />
        <param name="manhole_path" type="string" value="$(find hole_detection)/resource/manhole.txt"/>
        <param name ="big_hole_path" type="string" value="$(find hole_detection)/resource/bigHole.txt"/>
    </node> -->
</launch>